Makefile

```diff
@@ -49,3 +56,19 @@ dev:
 	@echo "Running with local development themes"
 	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} ${FLAGS}
 	python -m webbrowser -t "file://$(PWD)/render/index.html"
+
+# Running locally 
+# We optionally include our .env file if it exists (compatibility with the docker
+# --env-file) and then create a vitualenv as needed for running our local command.
+ifneq (,$(wildcard ./.env))
+    include .env
+    export
+endif
+
+$(PYTHON_BIN):
+	$(SYSTEM_PYTHON) -m venv $(VENV_DIR)
+	$(PYTHON_BIN) -m pip install -r requirements.txt
+
+.PHONY: local
+local: $(PYTHON_BIN)
+	$(PYTHON_BIN) run.py
```

Comment:
This can probably`${FLAGS}` just as the `run` target does.

Reply:
Fixed in a024fcfdd1325e35c150eab477fb0079474f9fdd